client/changelist/change.go

```diff
@@ -38,8 +38,10 @@ type TufChange struct {
 // TufRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TufRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	ReplaceKeys data.KeyList `json:"keys"`
```

Comment:
Rather than having 3 fields, it seems like `AddKeys` and `RemoveKeys` would be sufficient with differerent actions denoting how those fields get used.

At an extreme, we could even keep the single `Keys` field and separate out ordered actions for add and remove.